bot/resources/tags/path.md

```diff
@@ -0,0 +1,12 @@
+**What is the PATH?**
+
+The PATH is a special environment variable that holds mulitple directory paths. The files in these directories can be executed from anywhere in the command line, on any directory.
+
+**Why would I want to modify the PATH?**
+
+There are multiple reasons you would want to modify your PATH. One reason is to use applications installed in other directories without including the path to execute the application. Another reason would be to make scripts more easily accessible.
+
+**Resources**
+
+• [What the heck is my PATH, and how do I get it to do what I want?](https://astrobiomike.github.io/unix/modifying_your_path)
+•  [How do I set or change the PATH system variable?](https://java.com/en/download/help/path.html)
```

Comment:
What's the reason for the extra space at the beginning here?

Reply:
@Kronifer 
```suggestion
• [How do I set or change the PATH system variable?](https://java.com/en/download/help/path.html)
```